wandb/sdk/launch/builder/build.py

```diff
@@ -436,11 +436,12 @@ def join(split_command: List[str]) -> str:
     return " ".join(shlex.quote(arg.replace("'", "")) for arg in split_command)
 
 
-def construct_builder_args(
+def construct_agent_configs(
```

Comment:
the change here is to fix a bug that is not vertex specific. When launching a run without an agent by running
`wandb launch -j <job> -r <resource>` the `environment` block was not being loaded from the agent config.